profiles/fscloud/outputs.tf

```diff
@@ -0,0 +1,13 @@
+##############################################################################
+# Outputs
+##############################################################################
+
+output "id" {
+  description = "mongodb instance id"
+  value       = module.mongodb.id
+}
+
+output "version" {
+  description = "mongodb instance version"
+  value       = module.mongodb.version
+}
```

Comment:
This is confusing that the profile contains outputs and yet the example does not. One of the examples also outputs guid. Should that be included in the review? I am unclear how this all factors into the CRA checks.

Reply:
I believe users can alter examples when they use them. I have not sure about the CRA scan either but it doesn't look like having an output in the examples too. I will add them. In case there are issues with the CRA scan that can be addressed in the subsequent PR.

Reply:
Yes, the complete example has GUID in it, not sure about the business requirement though. For now, I have added "id" and "version" outputs in the FSCloud example, if need be will add "guid" as well.